server/router_utils.go

```diff
@@ -236,46 +236,40 @@ func readErrorKey(writer http.ResponseWriter, request *http.Request) (types.Erro
 	return types.ErrorKey(errorKey), nil
 }
 
-// readVoteOnRuleParams gets cluster_name, rule_id and user_id from current request
-func (server *HTTPServer) readVoteOnRuleParams(
+// readClusterRuleUserParams gets cluster_name, rule_id and user_id from current request
+func (server *HTTPServer) readClusterRuleUserParams(
 	writer http.ResponseWriter, request *http.Request,
-) (types.ClusterName, types.RuleID, types.UserID, bool) {
+) (types.ClusterName, types.RuleID, types.UserID, error) {
 	clusterID, err := readClusterName(writer, request)
 	if err != nil {
 		// everything has been handled already
-		return "", "", "", false
```

Comment:
why to return error when we're not using it anywhere?

Reply:
At first, I thought that the `bool` here is mainly for the `checkVotePermission` which I split, so I changed this function to be consistent with the rest of the functions in `router_utils`, they always return the errors even though everything has been handled already (logged + response sent) @Sergey1011010

Reply:
@Sergey1011010 I agree that it doesnt make any sense to return errors when they are already logged and handled. We can refactor this later and change all these "everything has been handled" errors to "ok" booleans, but for me, consistency is key